zebra-state/src/service/finalized_state/zebra_db/block/tests/snapshots/utxos@mainnet_0.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: zebra-state/src/service/finalized_state/zebra_db/block/tests/snapshot.rs
+expression: stored_utxos
+---
+[
+  (OutPoint(
+    hash: "c4eaa58879081de3c24a7b117ed2b28300e7ec4c4c1dff1d3f1268b7857a4ddb",
+    index: 0,
+  ), None),
```

Comment:
This is the UTXO for the genesis transparent output, which Zebra doesn't store, because of a consensus rule.